client/galaxy/scripts/components/Panels/ToolBox.vue

```diff
@@ -44,11 +57,14 @@ import ToolSection from "./Common/ToolSection";
 import ToolSearch from "./Common/ToolSearch";
 import UploadButton from "./Buttons/UploadButton";
 import FavoritesButton from "./Buttons/FavoritesButton";
-import { filterToolSections } from "./utilities";
+import { filterToolsinCats, filterTools, resizePanel } from "./utilities";
 import { getGalaxyInstance } from "app";
 import { getAppRoot } from "onload";
 import _l from "utils/localization";
 
+const EXPANDED_WIDTH = 400;
+const DEFAULT_WIDTH = 288;
```

Comment:
Can we move these into `css` and use percentages if possible?

Reply:
Not using the hardcoded numbers. Changed to calculated values based on the left and centre panels